src/components/Gallery/styled.ts

```diff
@@ -112,13 +113,18 @@ export const CarouselButton = styled(GalleryButton as any)`
 export const ImageWrapper = styled.div`
   text-align: center;
   width: 100%;
-  height: 100%;
-  display: inline-block;
+  height: auto;
+  display: flex;
+  flex-direction: column;
   transition-property: transform, visibility;
-
+  font-size: 0;
+  margin: auto;
   img {
+    aspect-ratio: ${({ aspectRatio }) => aspectRatio};
     object-fit: contain;
-    height: 100%;
+    height: 450px;
```

Comment:
did we test without the height? was this added intentionally?

Reply:
without height, it was defaulting to the smallest dimension locally (25vh) so I had a choice of making it 100%, 450px or some other value to make it stretch to a respectible size.  I figured the old value would be safe for compatibility.

Reply:
gotcha gotcha -- makes sense, just checking

Reply:
No Height: 
![image](https://user-images.githubusercontent.com/2287238/147783118-7f464dc4-257a-4917-87ee-30fc8fe26d5b.png)

Reply:
no height works fine in the page (interestingly enough), but storybook was not having it.

Reply:
But now that I think of it, I should not be building this for Storybook... I'm going to remove the height because it works on the page properly.